src/components/DefineCompany/YourCompany.vue

```diff
@@ -203,10 +216,36 @@ export default class YourCompany extends Mixins(DateMixin, EntityFilterMixin, Le
       : 'Unknown'
   }
 
-  /** Handle the company name change data */
-  private async nameChangeHandler (isSaved: boolean): Promise<any> {
-    // Handle name corrections vs a snapshot of the original IA here for UI indicators.
-    if (isSaved) this.companyNameChanges = false
+  /** Compare current to corrected data and update UI.  */
+  private async nameChangeHandler (type: CorrectionTypes): Promise<any> {
```

Comment:
I added a switch case here to handle the different comparisons based off what type of name correction occurred. Without a Named Company to test with, i was 100% sure if the comparison was going to be the same, so currently the switch cases are using the same comparison method, but this could change. So this may seem redundant now, but could be utilized when we introduce the third scenario and/or have some named companies to test with. If unneeded, will refactor in the near future.